test/test.js

```diff
@@ -120,7 +119,7 @@ test('casetest sample', t=>{
   note "example casetest jams"
   func adder
   args [1 2]
-  want ["" 3]
```

Comment:
Again `""` is a syntax error.

Reply:
Definitely we need `""` to be able to specify empty string so I wonder what is going wrong